src/components/Filters/Filters.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import FilterStyle from './FiltersStyle'
+import Button from '../../styledComponents/Button'
+import {setProductsSort} from './actionCreators'
+
+const mapStateTopProps = (state) => ({
+  sortType:state.sortProducts.sortProductsType
+})
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    setProductsSort: (sortType) => dispatch(setProductsSort(sortType))
+  }
+}
+const Filters = (props) => {
+    return (
+      <FilterStyle>
+        <span>Sort by price:</span>
+
+        <Button active={props.sortType === 'PRICE_ASC' ? 'active' : false}
```

Comment:
How about calling the <Button> with some less generic name? It's not very obvious it's your component, not basic html <button>